lib/genome/importers/hingorani_casas/hingorani_casas_tsv_importer.rb

```diff
@@ -0,0 +1,26 @@
+module Genome
+  module Importers
+    module HingoraniCasas
+      def self.source_info
+        {
+          base_url:           'http://www.ncbi.nlm.nih.gov/gene?term=',
+          site_url:           'http://stm.sciencemag.org/content/9/383/eaag1166',
+          citation:           'The druggable genome and support for target identification and validation in drug development. Finan C, Gaulton A, Kurger F, Lumbers T, Shah T, Engmann J, Galver L, Kelly R, Karlsson A, Santos R, Overington J, Hingorani A, Casas JP.   Sci. Transl. Med. 2017 Mar 29;9(383):eaag1166 PMID: 28356508.',
+          source_db_version:  '31-May-2017',
+          source_type_id:     DataModel::SourceType.POTENTIALLY_DRUGGABLE,
+          source_db_name:     'HingoraniCasas',
+          full_name:          'The druggable genome and support for target identification and validation in drug development (Hingorani & Cass, 2017)'
+        }
+      end
+
+      def self.run(tsv_path)
+        TSVImporter.import tsv_path, HingoraniCasasRow, source_info do
+          gene :gene_symbol, nomenclature: 'HingoraniCasas Gene Symbol' do
+            name :gene_symbol, nomenclature: 'Gene Symbol', transform: ->(x) { x.upcase }
+            name :ensembl_id, nomenclature: 'Ensembl Id', transform: ->(x) { x.upcase }
+          end
```

Comment:
Thanks @acoffman for taking look at this! I fixed that error and reran it. 
These are the messages I got this time. 

> Starting import!
> Skipping presumed header line in /Users/kcotto/Documents/Hingorani_Casas_genes...
> Imported 4479 gene_claims.
> Imported 8958 gene_claim_aliases.
> Imported 0 gene_claim_attributes.
> Imported 0 gene_gene_interaction_claims.
> Imported 0 gene_gene_interaction_claim_attributes.
> Imported 0 gene_claim_categories_gene_claims.
> Imported 0 drug_claims.
> Imported 0 drug_claim_aliases.
> Imported 0 drug_claim_attributes.
> Imported 0 interaction_claims.
> Imported 0 interaction_claim_attributes.
> Clearing cache
> Done.

When I look at my local database copy, I get a 'NoMethodError at /sources' webpage. I'm not sure if this is something I've done wrong or not.